src/envoy/auth/config.cc

```diff
@@ -158,6 +167,18 @@ IssuerInfo::IssuerInfo(Json::Object *json) {
   failed_ = true;
 }
 
+bool IssuerInfo::IsValidAudience(const std::string &aud) {
+  if (audiences_.empty()) {
+    return true;
+  }
+  for (auto &a : audiences_) {
```

Comment:
variable name `a` is not good, name it `audience`?

Reply:
resolved by using std::find()